BT-Tracking/Services/RemoteValues.swift

```diff
@@ -123,11 +123,16 @@ struct RemoteValues {
         return TimeInterval(AppDelegate.shared.remoteConfigInt(forKey: RemoteConfigValueKey.uploadWaitingMinutes) * 60 * 60)
     }
 
-    /// počet dní, jak dlouho se mají držet data v telefonu, default = 14
-    static var persistDataDays: TimeInterval {
-        return TimeInterval(AppDelegate.shared.remoteConfigInt(forKey: RemoteConfigValueKey.uploadWaitingMinutes) * 60 * 60 * 24)
+    /// počet dní, jak dlouho se mají držet data v telefonu jako TimeInterval, tedy v sekundách, default = 14 * 86400
+    static var persistDataInterval: TimeInterval {
+        return TimeInterval(AppDelegate.shared.remoteConfigInt(forKey: RemoteConfigValueKey.persistDataDays) * 60 * 60 * 24)
```

Comment:
To uz muzes rovnou zavolat tu funkci Self.persistDataDays misto toho mit tam 2x stejny remote config